src/ARSamples/SampleHelpers/DownloadManager/FileDownloadTask.cs

```diff
@@ -80,45 +80,10 @@ private FileDownloadTask(string filename, Uri requestUri, HttpResponseMessage co
 
         public event EventHandler<Exception>? Error;
 
-        [OnDeserialized]
-        internal void OnDeserialized(StreamingContext context)
-        {
-            if (Status == DownloadStatus.Downloading || Status == DownloadStatus.Resuming)
-                Status = DownloadStatus.Paused;
-            var fi = new FileInfo(Filename);
-
-            if (fi.Exists)
-            {
-                IsResumable = !string.IsNullOrEmpty(ETag) || Date.HasValue;
-                BytesDownloaded = fi.Length;
-            }
-        }
-
-        public static FileDownloadTask FromJson(string json, HttpMessageHandler? handler = null)
-        {
-            DataContractJsonSerializer s = new DataContractJsonSerializer(typeof(FileDownloadTask));
-            using (var ms = new MemoryStream(Encoding.UTF8.GetBytes(json)))
-            {
-                var fdt = (FileDownloadTask)s.ReadObject(ms)!;
-                fdt.client = new HttpClient(handler ?? new HttpClientHandler());
-                return fdt;
-            }
-        }
-
-        public string ToJson()
-        {
-            DataContractJsonSerializer s = new DataContractJsonSerializer(typeof(FileDownloadTask));
-            using (var ms = new MemoryStream())
-            {
-                s.WriteObject(ms, this);
-                return Encoding.UTF8.GetString(ms.ToArray());
-            }
-        }
-
```

Comment:
Bring back. Not sure why we don't want to use it any longer? Commit jsut says "Fix download manager to work with .net6-ios", but I only see removed code, no fix